alot/db/manager.py

```diff
@@ -385,6 +385,9 @@ def get_threads(self, querystring, sort='newest_first'):
         assert sort in self._sort_orders
         q = self.query(querystring)
         q.set_sort(self._sort_orders[sort])
+        if exclude_tags:
```

Comment:
Oh, but then we also need to use `fallback=()` in the `get_notmuch_settings` call. Not sure which is nicer ...